code/datums/diseases/transformation.dm

```diff
@@ -127,15 +127,18 @@
 	if(affected_mob.mind && !is_monkey(affected_mob.mind))
 		add_monkey(affected_mob.mind)
 	if(ishuman(affected_mob))
-		if(affected_mob && !is_monkey_leader(affected_mob.mind))
-			var/mob/living/carbon/monkey/M = affected_mob.monkeyize(TR_KEEPITEMS | TR_KEEPIMPLANTS | TR_KEEPORGANS | TR_KEEPDAMAGE | TR_KEEPVIRUS | TR_KEEPSE)
+		H.add_hud_to(new/datum/atom_hud/data/human/medical/advanced())
```

Comment:
No.
```suggestion
		GLOB.huds[DATA_HUD_MEDICAL_ADVANCED].add_hud_to(H)
```

Reply:
Et cetera. Don't instance new hud datums for every mob.

Reply:
Like such?